### ISSUE TITLE
code??

### ISSUE THREAD

**Issue by Maarij07** [Prabhat-2101/Ride-Karo#1]
is there any code??

**Comment by Prabhat-2101** [comment]
Yes, the code is available but it is not added in the repository. I am
trying to complete the same project using MERN as it was done using JAVA
and MYSQL previously.
